README.rst

```diff
@@ -67,19 +79,20 @@ then
 
 ``tox``
 
-It's also possible to create a ``./secrets.json`` as above, then::
+It's also possible to create a ``./secrets.json`` as above, and `build the docker image`_. Then::
```

Comment:
It appears that creating a secrets.json is *required* to build the docker image.